common/libs/wallet/base.js

```diff
@@ -1,6 +1,6 @@
 // @flow
-import { stripHex } from 'libs/values';
-import type { RawTransaction } from 'libs/transaction';
+import { stripHex } from '../values';
+import type { RawTransaction } from '../transaction';
```

Comment:
I'm not sure if it would be in addition to, or in lieu of, but there's a babel plugin for this: https://www.npmjs.com/package/babel-plugin-module-resolver